x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/IndexFollowingIT.java

```diff
@@ -101,9 +102,24 @@ public void testFollowIndex() throws Exception {
         assertAcked(leaderClient().admin().indices().prepareCreate("index1").setSource(leaderIndexSettings, XContentType.JSON));
         ensureLeaderYellow("index1");
 
-        final int firstBatchNumDocs = randomIntBetween(2, 64);
+        final int firstBatchNumDocs = randomIntBetween(800, 1200);
+        final int flushPoint = (int) (firstBatchNumDocs * 0.75);
+
         logger.info("Indexing [{}] docs as first batch", firstBatchNumDocs);
-        for (int i = 0; i < firstBatchNumDocs; i++) {
+        BulkRequestBuilder bulkRequestBuilder = leaderClient().prepareBulk();
+        for (int i = 0; i < flushPoint; i++) {
+            final String source = String.format(Locale.ROOT, "{\"f\":%d}", i);
+            IndexRequest indexRequest = new IndexRequest("index1", "doc", Integer.toString(i))
+                .source(source, XContentType.JSON)
+                .timeout(TimeValue.timeValueSeconds(1));
+            bulkRequestBuilder.add(indexRequest);
+        }
+        bulkRequestBuilder.get();
+
+        leaderClient().admin().indices().prepareFlush("index1").setForce(true).setWaitIfOngoing(true).get();
+
+        // Index some docs after the flush that will be recovered in the normal index following operations
```

Comment:
this comment is not true. There is no guarantee that these won't be flushed. Change the wording from `will` to `might`